File: vortexdbg-api/src/main/java/com/vortexdbg/listener/TraceWriteListener.java
package com.vortexdbg.listener;

import com.vortexdbg.Emulator;

public interface TraceWriteListener {

    /**
     * @return 返回<code>true</code>打印内存信息
     */
    boolean onWrite(Emulator<?> emulator, long address, int size, long value);

}

File: vortexdbg-api/src/main/kotlin/com/vortexdbg/listener/TraceWriteListener.kt
package com.vortexdbg.listener

import com.vortexdbg.Emulator

interface TraceWriteListener {

    /**
     * @return `true` to let the tracer print the memory access information
     */
    fun onWrite(emulator: Emulator<*>, address: Long, size: Int, value: Long): Boolean

}
